quickstarts/microsoft.compute/encrypt-running-vmss-linux/azuredeploy.json

```diff
@@ -48,22 +48,34 @@
       "metadata": {
         "description": "Pass in a unique value like a GUID everytime the operation needs to be force run"
       }
-    }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description":"Default location"
+      }
+    },
+    "valuturi": {
```

Comment:
remove this and use the environment() function to get the uri

Reply:
updated with environment() function